python/config_settings/BUILD.bazel

```diff
@@ -92,6 +92,24 @@ string_flag(
     visibility = ["//visibility:public"],
 )
 
+string_flag(
+    name = "py_freethreaded",
```

Comment:
This is similar to `py_linux_libc`, but we could have other names as well. Suggestions?

Reply:
I think py_freethreaded SGTM